velox/exec/PartitionedOutputBufferManager.h

```diff
@@ -144,31 +197,63 @@ class PartitionedOutputBuffer {
   /// updating the total number of drivers, we don't.
   void checkIfDone(bool oneDriverFinished);
 
-  // Updates buffered size and returns possibly continuable producer promises in
-  // 'promises'.
+  // Updates buffered size and returns possibly continuable producer promises
+  // in 'promises'.
   void updateAfterAcknowledgeLocked(
       const std::vector<std::shared_ptr<SerializedPage>>& freed,
       std::vector<ContinuePromise>& promises);
 
   /// Given an updated total number of broadcast buffers, add any missing ones
   /// and enqueue data that has been produced so far (e.g. dataToBroadcast_).
-  void addBroadcastOutputBuffersLocked(int numBuffers);
+  void addOutputBuffersLocked(int numBuffers);
+
+  void enqueueBroadcastOutputLocked(
```

Comment:
Same question: why can't we have a single enqueueOutputLocked method?

Reply:
Updated